Generating C executable that accesses the shared object file without Python (#1003)
PennyLaneAI/catalyst: doc/changelog.md

      %3 = quantum.expval %2 : f64
  ...
  }

> 

Reviewer: ```suggestion
```

   8.0
  ```

> * Catalyst now supports c executable generation with `catalyst.debug.compiler_functions.compile_executable`.

Reviewer: Same as in the other PR :) 
```suggestion
* Catalyst now supports c executable generation with `catalyst.debug.compile_executable`.
```

>   It also supports mutti-dimensional arrays as function's inputs. [(#1003)](https://github.com/PennyLaneAI/catalyst/pull/1003)

Reviewer: ```suggestion
  A bug is fixed in `catalyst.debug.get_cmain` to support multi-dimensional arrays as
  function inputs. 
  [(#1003)](https://github.com/PennyLaneAI/catalyst/pull/1003)
```
